bool use_current_info

# If false then fill out the details

string filename
string name
string description
int32 level
---
bool status
string message
